### ISSUE TITLE
Modules consistency false-positive with EXM 3.5.0

### ISSUE THREAD

**Issue by AlenPelin** [Sitecore/Sitecore-Diagnostics-Tool#41]
Sitecore 8.2.5 and EXM 3.5.0 give false-positive note:
```
An inconsistent module was detected: Web Forms for Marketers.
```
